ci-operator/step-registry/openshift/microshift/infra/aws/ec2/openshift-microshift-infra-aws-ec2-commands.sh

```diff
@@ -44,8 +26,20 @@ echo -e "AMI ID: $ami_id"
 cat > "${cf_tpl_file}" << EOF
 AWSTemplateFormatVersion: 2010-09-09
 Description: Template for RHEL machine Launch
-
+Conditions:
+  AddSecondaryVolume: !Not [!Equals [!Ref EC2Type, 'MetalMachine']]
+Mappings:
+ VolumeSize:
+   MetalMachine:
+     PrimaryVolumeSize: "300"
+     SecondaryVolumeSize: "0"
+   VirtualMachine:
+     PrimaryVolumeSize: "200"
+     SecondaryVolumeSize: "100"
```

Comment:
I would rather go with 20+10 sizes here. Do we need so much disk space for e2e tests?